tests/primer/primer_tool.py

```diff
@@ -88,6 +100,35 @@ def _handle_prepare_command(self) -> None:
             ) as f:
                 f.write(commit_string)
 
+    def _handle_run_command(self) -> None:
+        if Path(PRIMER_DIRECTORY / f"output_{self.config.type}.txt").exists():
+            os.remove(PRIMER_DIRECTORY / f"output_{self.config.type}.txt")
```

Comment:
I guess this is what confused me -- if this isn't executing on the CI runners, is this for local testing? If so, should we just remove this and do "w" mode below?